media_kit/lib/src/player/platform_player.dart

```diff
@@ -40,19 +39,18 @@ abstract class PlatformPlayer {
   late PlayerState state = PlayerState();
 
   /// Current state of the player available as listenable [Stream]s.
-  late PlayerStreams streams = PlayerStreams(
+  late PlayerStream stream = PlayerStream(
```

Comment:
NOTE: `Player.streams` is now `Player.stream` to match the `Player.state` getter.

Old getter is kept with `@Deprecated` decorator.